src/scripts/util/data.model.js

```diff
@@ -63,11 +64,30 @@ function timeDomain(model){
 }
 
 function valueDomain(model){
-	if (model.valueDomain) { return model.valueDomain; }
-	var extents = setExtents(model);
-	var domain = d3.extent(extents);
-	if (!model.falseOrigin && domain[0] > 0) {
-		domain[0] = 0;
+	if(model.valueDomain){return model.valueDomain;}
+
+	var domain, extents;
+	if(model.chartSubtype !== 'stacked'){
+		extents = setExtents(model);
+		domain = d3.extent(extents);
+		if(!model.falseOrigin && domain[0] > 0){
+			domain[0] = 0; //see line 88
+		}
+	}else{
+		var vals = [];
+
+		model.data.map(function (d, j){
+			var k, s = 0;
+			for(k in d.values[0]){
+				if(k !== 'date'){
+					s += d.values[0][k];
+				}
+			}
+			vals.push(s);
+		});
```

Comment:
Maybe this kind of data manipulation should exist better in nightingale? 

E.g. if it is necessary to push the data within this data structure then the manipulation should happen at the point of import?